about.php

```diff
@@ -47,8 +47,8 @@
 
 	<div class="heritage">
 		<h3>Our Heritage</h3>
-		<p>We are humbled by the success of our company thus far. The name Starbulls was created by one of the founders:
-			Will. He came up with this insane concept of a restaurant that serves wings and coffee.
+		<p>We are humbled by the success of our company thus far. The name StarBulls was created by one of the founders,
```

Comment:
...created by Will, one of our founders.

Reply:
adjusted